guides/common/modules/ref_preparing-smartproxyservers-for-load-balancing-next-steps.adoc

```diff
@@ -0,0 +1,4 @@
+[id="preparing-{smart-proxy-context}-servers-for-load-balancing-next-steps_{context}"]
+= Next steps
```

Comment:
Any reason to not go with ".Next steps"? same above for "additional resources".

Reply:
The headings and therefore the navigation looks a bit off (very short and generic), but I totally get your reasoning and I'm OK with it.

Reply:
Thanks for voicing your opinion @maximiliankolb I get it, I agree, but I don't have a better solution that would work for what I'm trying to do. I'm happy to revisit this in the future if we come up with something better.